2996_reflection/reflection.md

```diff
@@ -4030,6 +4119,24 @@ Add a row to [tab:dcl.type.simple] to cover the `$splice-type-specifier$` produc
 |`...`|...|
 :::
 
+### [dcl.type.decltype]{.sref} Decltype specifiers {-}
+
+Add a bullet after bullet 1.3 to apply to `$splice-expression$`s:
+
+::: std
+[1]{.pnum} For an expression `$E$`, the type denoted by `decltype($E$)` is defined as follows:
+
+[...]
+
+- [1.3]{.pnum} otherwise, if `$E$` is an unparenthesized `$id-expression$` or an unparenthesized class member access ([expr.ref]), `decltype($E$)` is the type of the entity named by `$E$`. If there is no such entity, the program is ill-formed;
+- [[1.3+]{.pnum} otherwise, if `$E$` is an unparenthesized `$splice-expression$`, `decltype($E$)` is the type of the entity, object, or value named by `$E$`;]{.addu}
```

Comment:
Is the only reason this isn't folded into the previous bullet that there it's "entity named by E" and here it's "entity, object, or value named by E"?

Reply:
Ah, not quite - it should be "designated the _splice-specifier_ of" not "named by". Thanks.